src/test/groovy/graphql/introspection/GoodFaithIntrospectionInstrumentationTest.groovy

```diff
@@ -16,6 +19,18 @@ class GoodFaithIntrospectionInstrumentationTest extends Specification {
         GoodFaithIntrospection.enabledJvmWide(true)
     }
 
+    def "standard introspection query is inside limits just in general"() {
+
+        when:
+        Document document = TestUtil.toDocument(IntrospectionQuery.INTROSPECTION_QUERY)
+        def eno = ExecutableNormalizedOperationFactory.createExecutableNormalizedOperation(graphql.getGraphQLSchema(),document,
+        "IntrospectionQuery", CoercedVariables.emptyVariables())
+
+        then:
+        eno.getOperationFieldCount() < GoodFaithIntrospection.GOOD_FAITH_MAX_FIELDS_COUNT  // currently 62
+        eno.getOperationDepth() < GoodFaithIntrospection.GOOD_FAITH_MAX_DEPTH_COUNT  // currently 13
+    }
```

Comment:
a test to make sure our default introspection query stays within good faith limits